src/Context.cpp

```diff
@@ -216,6 +216,9 @@ void Context::InitResourceManager(const std::vector<std::string>& otrFiles,
         SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, "OTR file not found",
                                  "Main OTR file not found. Please generate one", nullptr);
         SPDLOG_ERROR("Main OTR file not found!");
+#ifdef PLATFORM_IOS
+        exit(0);
+#endif
```

Comment:
couple things

why `PLATFORM_IOS` as opposed to the patterns we have for other things like `__SWITCH__` - not a problem just curious

it'd be nice to add a comment explaining why exit 0 is needed for ios here